### PR TITLE
chore: add how to guide for add tab

### PR DESCRIPTION
**Pull request by hund030**
image

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -400,6 +400,8 @@
     "teamstoolkit.tutorials.connectApi.label": "Connect to an API",
     "teamstoolkit.tutorials.addAzureSql.label": "Integrate with Azure SQL Database",
     "teamstoolkit.tutorials.addAzureSql.detail": "A platform as a service (PaaS) database engine to serve as your Teams applications backend.",
+    "teamstoolkit.tutorials.addTab.label": "Configure Tab capability",
+    "teamstoolkit.tutorials.addTab.detail": "Teams identity aware webpages embedded in Microsoft Teams",
```
Comment (timngmsft) [inline packages/vscode-extension/package.nls.json:404]: `identity-aware` i think is more appropriate

Comment (hund030) [reply packages/vscode-extension/package.nls.json]: Updated.

**Comment by MuyangAmigo** [comment]
@hund030 can you include a screenshot?

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -400,6 +400,8 @@
     "teamstoolkit.tutorials.connectApi.label": "Connect to an API",
     "teamstoolkit.tutorials.addAzureSql.label": "Integrate with Azure SQL Database",
     "teamstoolkit.tutorials.addAzureSql.detail": "A platform as a service (PaaS) database engine to serve as your Teams applications backend.",
+    "teamstoolkit.tutorials.addTab.label": "Configure Tab capability",
+    "teamstoolkit.tutorials.addTab.detail": "Teams identity-aware webpages embedded in Microsoft Teams",
```
Comment (MuyangAmigo) [inline packages/vscode-extension/package.nls.json:404]: Desc: Teams-aware webpages embedded in Microsoft Teams.

Comment (MuyangAmigo) [inline packages/vscode-extension/package.nls.json:404]: Tab -> tab

Comment (hund030) [reply packages/vscode-extension/package.nls.json]: Updated.

Comment (hund030) [reply packages/vscode-extension/package.nls.json]: Updated.

**Pull request closed without merging**